Unified/htmlor.py

```diff
@@ -19,7 +19,7 @@ def htmlor( caller = ""):
 
     for backup in ['statuses.json','siteInfo.json','equalizor.json']:
         print "copying",backup,"to old location"
-        os.system('cp %s/%s /afs/cern.ch/user/c/cmst2/www/unified/.'%(monitor_pub_dir, backup))
+        os.system('env EOS_MGM_URL=root://eoscms.cern.ch eos cp %s/%s /afs/cern.ch/user/c/cmst2/www/unified/.'%(monitor_pub_dir, backup))
```

Comment:
I don't think this is eos cp as it's doing cp in afs and not eos directory.

Reply:
https://github.com/CMSCompOps/WmAgentScripts/blob/master/utils.py#L97
I see it's actually using eos here.